src/plugins/plugins.js

```diff
@@ -57,9 +57,19 @@ define([
         };
     };
 
-    plugins.AutoflowView = function () {
-        return AutoflowPlugin;
-    };
+    /**
+     * @typedef {Object} AutoflowOptions
+     * @property {string} [type] The key of an object type to apply this view
+     * to exclusively.
+     */
+    /**
+     * A tabular view showing the latest values of multiple telemetry points at
+     * once. Formatted so that labels and values are aligned.
+     *
+     * @param {AutoflowOptions} [options] Optional settings to apply to the autoflow
```

Comment:
@akhenry Instead of defining a type just to describe options, you can do:

```jsdoc
@param {Object} [options] options for the autoflow view.
@param {string} [options.type] restrict autoflow to this type of object.
```